Modules/Core/Common/include/itkDefaultConvertPixelTraits.h

```diff
@@ -140,7 +140,7 @@ class DefaultConvertPixelTraits< Offset< VDimension > >
 
 #define ITK_DEFAULTCONVERTTRAITS_FIXEDARRAY_TYPE(type)                  \
   template<typename TComponentType, unsigned VDimension >               \
-  class DefaultConvertPixelTraits< type< TComponentType, VDimension > > \
+  class ITK_TEMPLATE_EXPORT DefaultConvertPixelTraits< type< TComponentType, VDimension > > \
```

Comment:
Macro continuation character alignment

Reply:
Meh, that is a lot of line of code churning to align the continuation character here....